libs/react-spa/bff/src/lib/BffProvider.tsx

```diff
@@ -1,5 +1,12 @@
 import { useEffectOnce } from '@island.is/react-spa/shared'
-import { ReactNode, useCallback, useEffect, useReducer, useState } from 'react'
+import {
```

Comment:
Let's also update the delegation email notification URL. 👍

Reply:
Then I will need to disable  the test if you are referring to this 
https://github.com/island-is/island.is/blob/55aacf7935422425da140c2e254d3d25fdeb2660/apps/services/user-notification/src/app/modules/notifications/notificationsWorker/notificationsWorker.service.ts#L479

See test here
https://github.com/island-is/island.is/blob/55aacf7935422425da140c2e254d3d25fdeb2660/apps/services/user-notification/src/app/modules/notifications/notificationsWorker/notificationsWorker.service.spec.ts#L306